### DISCUSSION TITLE
Software interface display abnormality

### DISCUSSION THREAD

**Discussion by baobao418** [freeplane/freeplane#1132] [Help]
It seems this problem only happened in my PC maybe:( . My version is 1.11.2
when I create a new map, after adding a new child node on the basic "New mindmap"node and keying in some words, this problem will happen.
<img width="1015" alt="屏幕截图 2023-05-10 171038" src="https://github.com/freeplane/freeplane/assets/111677450/fa83a786-c3cb-4d34-874a-1bffd438d093">
<img width="1015" alt="屏幕截图 2023-05-10 172526" src="https://github.com/freeplane/freeplane/assets/111677450/d86756d1-55e6-4072-8681-3ead083a6d2f">

I dont have any idea about what can cause this problem, and I dont see anyone else who also reported this problem.
It's great helpful to me if anyone can figure it out.
Thanks.

**Comment by dpolivaev** [comment]
I have seen this issue also on the computer of my friend living abroad, but I have no idea what makes it happen.
It looks like some graphics card or driver problem.
So I converted this bug report to a discussion so that the community can help with ideas.

**Reply by baobao418** [reply]
Thanks, I will seek more ways to figure this problem.

**Comment by EdoFro** [comment]
@baobao418 
I have had errors like that after a script had an error. 
maybe it would help to take a look to the log file. 

do you use any conditional format with script rules?
do you have any add-on installed?

does Freeplane start with the error when fresh opening?

**Reply by baobao418** [reply]
This problem will happen in minutes when I create a new mind map and its new child note as well as I open Freeplane, and this problem stop me doing anything but shut it down.
No add-on or conditional format in the application.

**Reply by ClaytonCurtis1947** [reply]
What OS are you running?  (Since you said "PC", I'm going to assume some version of Windows.)
Are you running in an unusual setting? E.g., virtual machine, atypical graphics card, etc.
I'm running 1.11.4 on Windows 10 and do not have this problem.

I've seen =similar= behavior in other contexts. Usually as a result of some form of corruption, but I'm not entering that swamp here!! :)  Out of date or buggy video drivers can certainly cause things like this. You might insure that you are running the latest ones.  One thing that =suggests= that is the duplication of parts of the user interface.

If you have access to another PC, the simplest initial troubleshooting step would be to install the same version on that PC and see if the behavior can be duplicated. (I suspect you already know that -- sorry.)

**Reply by ClaytonCurtis1947** [reply]
To dip a toe into the swamp ... If you haven't already, run (or find someone who can run -- these steps aren't trivial) CHKDSK (check disk for logical / structural errors) and SFC (system file checker); if SFC finds problems that it can't correct, someone has to try to run DISM (Deployment Image Service and Management -- YIKES). One =might= also check the system Event Viewer for something that even =appears= to relate or has an entry created every time you try your scenario.

**Comment by chandansomani** [comment]
Facing same issue 
my version 1.11.6

Bug not yet fixed?..

![image](https://github.com/freeplane/freeplane/assets/23182738/853b342e-ed20-4f08-bebe-883fda8aa309)

**Comment by Nunnusnunnu** [comment]
Hello,

I was experiencing this issue and ran through all the normal kind of troubleshooting steps, and eventually arrived at a solution:

**Go to tools, preferences, look and feel and select the default look and feel option. I was using Flat IntelliJ and it was causing this issue.**

I noticed this was happening in a reproducible way when I'd interact with the UI in any way. I could grab the data in the document and move it around, but once I actually touched a menu like File/Edit/Insert, whether by mouse or just keyboard, the bug would start instantly. I also noticed that when I'd switch to a single monitor (I typically use 3-4), the program would immediately crash upon switching back to a single monitor. It also seemed to be heavily triggered by some of my reminders that were overdue, maybe because they generate their own popup interface that follows the mouse?

Anyway, for anybody interested in fixing it or addressing it, here are some of the details of my environment for your reference if it's helpful at all:

Freeplane version 1.11.14 
Windows 11 Pro 64-bit (10.0, Build 22621)
Java Version 8 Update 411 (Build 1.8.0_411-b09)
AMD Adrenalin 24.5.1

Additional information:

This bug occurred even after I reinstalled Freeplane and cleared out any custom addons/icons/settings that I had. It also remains fixed after I pasted back my user/appdata folder and restored all my original settings (minus the change/fix to the default UI).

Thank you for your continued development and attention on the Freeplane project. I can say that it has been and remains a life altering application and I am grateful to have been able to use it for so many years.

**Reply by euu2021** [reply]
I think it confirms my hypothesis at https://github.com/freeplane/freeplane/issues/1348#issuecomment-2018968584
